web-components/src/[sandbox]/sandbox.ts

```diff
@@ -415,10 +407,10 @@ export class Sandbox extends LitElement {
             </sass-stats>
           </div>
 
-          <div class="container" aria-label="md-table">
-            <h2>md-table</h2>
-            <sass-stats component="table">
-              ${tableTemplate}
```

Comment:
Here too, seems unrelated to ComboBox, but not clear if it is cleanup or accidental removal

Reply:
@yananym fixed